src/redux/actionCreators.ts

```diff
@@ -174,6 +177,17 @@ export const setNodeEditorOpen = (nodeEditorOpen: boolean) => (
     }
 };
 
+export const applyUpdateSpec = (updateSpec: any) => (
```

Comment:
i think an `undefined` comparison might be more appropriate here, but also wondering if it isn't better for us to avoid `undefined/null/false`  comparisons in JavaScript where possible (given all the weirdness that follows). how do you feel about using a default parameter as a way to avoid that?

```
export const applyUpdateSpec = (updateSpec: any = {}) => (
	dispatch: DispatchWithState,
	getState: GetState
) => {
	if (Object.keys(updateSpec).length > 0) {
		const updatedDefinition = update(getState().definition, updateSpec);
		dispatch(updateDefinition(updatedDefinition)); 
	}
};
```